specification/2.0/Makefile

```diff
@@ -8,19 +8,18 @@ all: $(TARGETS)
 
 # Generate the Properties Reference section of the spec from JSON schema.
 # This uses 'npx' to minimally install the 'wetzel' tool, if needed.
-# While the schema used by wetzel is the locally installed version, it
-# generates inline links to the schema under $(SCHEMALINK), typically
-# the canonical schema in the github default branch.
+# When using embedded schemas, SCHEMALINK should be a relative path.
 
 WETZEL = npx wetzel
-SCHEMALINK = https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema
+SCHEMALINK = schema
+EMBEDSCHEMA = JsonSchemaReference.adoc
 
 # Base name of the generated properties reference file
 PROPREF = PropertiesReference
 # Schema files to leave out of $(PROPREF)
 IGNORESCHEMA = '["gltfchildofrootproperty.schema.json", "gltfid.schema.json", "gltfproperty.schema.json"]'
 $(PROPREF).adoc: $(wildcard schema/*.json)
```

Comment:
Would add $(EMBEDSCHEMA) to targets here. Might want some sort of npx dependency too, but I'm not sure how that would work since the wetzel files are pulled from a remote.

Reply:
I think in the next round of changes I'm going to add the version number onto the end of wetzel.  Specifically, something like this:

```
WETZEL = npx wetzel@0.1.0-dev.1.2
```

That way, updates to new versions of wetzel must be manually done in the Makefile, not just pulled in automatically whenever someone publishes.  This would provide more stability of the history, for example pulling an old commit and re-running the old documentation would use the specific version of wetzel available at that point in history, and should exactly regenerate what the docs were back then.  Likewise, changes to the docs introduced by new wetzel versions could be found with bisect, etc.